docs/data/material/components/progress/CircularSize.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+import Stack from '@mui/material/Stack';
+import CircularProgress from '@mui/material/CircularProgress';
+
+export default function CircularSize() {
+  return (
+    <Stack spacing={2} direction="row" alignItems="center">
+      <CircularProgress size={'30px'} />
+      <CircularProgress size={40} />
+      <CircularProgress size={'3rem'} />
```

Comment:
No need to brackets:
```suggestion
      <CircularProgress size="30px" />
      <CircularProgress size={40} />
      <CircularProgress size="3rem" />
```